pkg/printers/humanreadable.go

```diff
@@ -26,20 +26,27 @@ import (
 	"text/tabwriter"
 
 	"github.com/fatih/camelcase"
-	"github.com/golang/glog"
 
 	"k8s.io/apimachinery/pkg/api/meta"
+	metav1alpha1 "k8s.io/apimachinery/pkg/apis/meta/v1alpha1"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
+	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/kubernetes/pkg/util/slice"
 )
 
+type PrintHandler interface {
+	Handler(columns, columnsWithWide []string, printFunc interface{}) error
```

Comment:
It's because the functions are strongly typed.  I think it's correct enough, we need to strongly validate these in the registration method which I plan on adding eventually.